### DISCUSSION TITLE
Rate limits

### DISCUSSION THREAD

**Discussion by AntonyKing7D** [drakkan/sftpgo#647] [Q&A] [answered]
Are there any default rate limits ? I'm running a load test at the moment, and my load test server keeps ending up in the defender blocklist. I'm hitting it at about 30rps , split over 2 nodes (ie, 15rps per sftpgo instance). I have defender enabled with a 5 minute ban time, but nothing else set for rate limiting.

**Comment by drakkan** [comment, marked as answer]
Hi,

perhaps your server exceeds the configured [max connection per host limit](https://github.com/drakkan/sftpgo/blob/main/sftpgo.json#L19) and so it generates [score_limit_exceeded events](https://github.com/drakkan/sftpgo/blob/main/docs/defender.md). If this is the case you should see this [log](https://github.com/drakkan/sftpgo/blob/main/common/common.go#L840)

**Reply by AntonyKing7D** [reply]
Thanks; I see the default is 20 for max conenctions per host.
